SPECS/mysql/mysql.spec

```diff
@@ -87,8 +87,15 @@ sudo -u test %make_build CTEST_OUTPUT_ON_FAILURE=1 test
 %{_libdir}/*.a
 %{_includedir}/*
 %{_libdir}/pkgconfig/mysqlclient.pc
+/usr/lib/private/icudt69l/brkitr/*.res
```

Comment:
Just curious, what are these new files from the update? And do they need to be included in the mysql-devel package?

Reply:
Yes, these files came with the new version update. I put them in devel because from the file extension types, I judged they don't need to be in the mysql package

Reply:
As discussed offline, will keep these files in the devel package for now. Mysql added these files in this commit https://github.com/mysql/mysql-server/commit/b3e00fe93bd5f5176f73788021ef4624c88d766d